components/MonitoringCommentFrame.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div>
+    <div class="MonitoringCommentFrame" :style="{ color: `${colors[level]}` }">
```

Comment:
たしかにこれでもうまくいくのですけど、borderにカラー指定したほうがよりシンプルだと思います。
```suggestion
    <div class="MonitoringCommentFrame" :style="{ borderColor: `${colors[level]}` }">
```